TestSwitchApi/TestSwitchApi/Controllers/CandidateController.cs

```diff
@@ -28,5 +30,20 @@ public ActionResult<CandidateListResponse> GetCandidates([FromQuery] PageRequest
             var candidateCount = _candidates.Count(pageRequest);
             return CandidateListResponse.Create(pageRequest, candidates, candidateCount);
         }
+
+        [HttpGet("{candidateId}")]
+        public ActionResult<CandidateTestResponseModel> GetCandidateTestSubmissions(int candidateId)
+        {
+            var candidate = _candidates.GetCandidateById(candidateId);
+            var submissions = _submissions.GetSubmissionsById(candidateId);
+            return new CandidateTestResponseModel(submissions, candidate);
+        }
+
+        /*[HttpGet("{candidateId}/test-submissions")]
+        public ActionResult<IEnumerable<CandidateTestModel>> GetTestSubmissions(int candidateId)
+        {
+            var submissions = _submissions.GetSubmissionsById(candidateId);
+            return submissions.ToList();
+        }*/
```

Comment:
Could you remove the commented out code? :)

Reply:
Commented out code removed